packages/core/crates/core-misc/src/constants.rs

```diff
@@ -0,0 +1,5 @@
+pub const DEFAULT_HEARTBEAT_INTERVAL: u32 = 60000;
```

Comment:
Public constants should have documentation on them, will help to generate docs.

Reply:
changed